plugin/normalize.go

```diff
@@ -158,11 +158,16 @@ func SplitHostPort(s string) (hosts []string, port string, err error) {
 	}
 
 	// Check if it parses as a reverse zone, if so we use that. Must be fully specified IP and mask.
-	_, n, err := net.ParseCIDR(s)
+	ip, n, err := net.ParseCIDR(s)
 	if err != nil {
 		return []string{s}, port, nil
 	}
-
+	if ip.To4() == nil { // v6 address, if the mask if not on a octet, it's not a valid cidr (we don't need to split v6 like v4)
+		ones, _ := n.Mask.Size()
+		if ones%8 != 0 {
+			return nil, "", fmt.Errorf("non-octet boundary is not supported for a v6 reverse: %s", ip)
```

Comment:
For ipv6, reverse zones are divided into segments of 4 bits (one hex per segment).
So, should this be `if ones%4 != 0 {` ... and `"non-quartet boundary is not supported`

Reply:
you're right! Also https://www.ripe.net/manage-ips-and-asns/db/support/configuring-reverse-dns uses /29 which negates this whole PR. Let me put the example in there in a test.

It may well be that 5d80a6e2 fixed the crash in go-cidr and this is now some superficial flavoring on top of it